writing-apps/our-first-app/packaging.md

```diff
@@ -62,3 +62,29 @@ This tells Flatpak Builder to build the manifest we just wrote into a clean `bui
 
 That wasn't too bad, right? We'll set up more complicated packaging in the future, but this is all that is required to submit your app to AppCenter Dashboard for it to be built, packaged, and distributed. If you'd like you can always read [more about Flatpak](https://docs.flatpak.org/en/latest/introduction.html).
 
+## Uninstalling the application
+
+Ninja and Flatpak both provide commands to uninstalling the application.  It is recommended to use the provided method in both cases.
+
+To remove our application with Flatpak, we will use Flatpak's `remove` command:
+
+```bash
+flatpak remove com.github.yourusername.yourrepositoryname
+```
+
+Flatpak will prompt you to remove your application.
+
+```bash
+flatpak remove com.github.yourusername.yourrepositoryname
```

Comment:
I suggest going with `uninstall` command here, for a couple of reasons:

1. The `remove` option [is just an alias](https://github.com/flatpak/flatpak/blob/main/app/flatpak-main.c#L82-L83) for the `uninstall` command.
1. [Flatpak documentation](https://docs.flatpak.org/en/latest/flatpak-command-reference.html#flatpak-uninstall) only mentions the uninstall command
1.  Ninja also uses the uninstall terminology, so it would be consistent.

Plus, I suggest adding the `--delete-data` flag to the command, as leftover app data is unlikely to be useful, and might lead to complications in development.

These 2 changes would turn this line into:

```suggestion
flatpak uninstall com.github.yourusername.yourrepositoryname --delete-data
```